pkg/controller/inference/playground_controller.go

```diff
@@ -121,7 +121,7 @@ func (r *PlaygroundReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 func (r *PlaygroundReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&inferenceapi.Playground{}).
-		Watches(&inferenceapi.Service{}, &handler.EnqueueRequestForObject{},
+		Watches(&inferenceapi.Service{}, handler.EnqueueRequestForOwner(r.Scheme, r.RESTMapper(), &inferenceapi.Playground{}, handler.OnlyControllerOwner()),
```

Comment:
Inference service can be created manually, so maybe this is not right? Playground is for quick start, once people has forked inference backend, like vllm-on-ascend-chip, playground couldn't support such backends, but inference could.

Reply:
If the inference service is mutually created by users or external controllers, Can the playground own the inference service? 

- https://github.com/carlory/llmaz/blob/9c77c3814349fa470ad198796d52223a765a3c08/pkg/controller/inference/playground_controller.go#L98

Reply:
Then there's no Playground, so 
- Playground is for quick start, it's optional, if not found, skip.
- Inference Service is for advanced deployment, it's the real service.

Reply:
1. A user (Bob) creates an Inference Service (foo)
2. Another user (Li) doesn't know the foo service already exists, then creates a Playground with the same name.
3. The service foo will be owned by the Playground, and the service foo may be deleted by Li via the Playground.

Reply:
We may need to handle this case when reconcile Playground

Reply:
Good catch, then the Playground is illegal, but seems we can't block the Playground creation, not supported in webhook, we may need to add another condition in Playground to report the error, what do you think? 
```
		condition := metav1.Condition{
			Type:    inferenceapi.PlaygroundProgressing,
			Status:  metav1.ConditionFalse,
			Reason:  "AbortProcessing",
			Message: "Playground owns the same name with a Service",
		}
```